app/scripts/controllers/overview.js

```diff
@@ -1132,9 +1134,16 @@ function OverviewController($scope,
   // extract & share
   var sortServiceInstances = function() {
     if(!state.serviceInstances && !state.serviceClasses) {
+      state.bindableServiceInstances = null;
       return;
     }
-    state.orderedServiceInstances = _.toArray(state.serviceInstances).sort(function(left, right) {
+
```

Comment:
Curious, rather than creating a duplicate array `serviceInstancesArray`, `_.filter` works fine on a hash as does `_.sortBy` instead of `[].sort`.  Possible to use the `_` version?